src/screens/OneOnOneScreen/OneOnOneScreen.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react';
+import { object } from 'prop-types';
+import { View, FlatList } from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+import { useStatus, SUCCESS } from '@rootstrap/redux-tools';
+
+import { getOneOnOne } from 'actions/oneOnOneActions';
+
+import { Loader, Text } from 'components';
+import OneOnOneInformation from 'components/OneOnOneInformation/oneOnOneInformation';
+
+import ActionItem from 'components/ActionItem/ActionItem';
+import useAlertError from 'hooks/useAlertError';
+import styles from './OneOnOneScreen.styles';
+
+const OneOnOneScreen = ({
+  navigation,
+  route: {
+    params: { idOneOnOne },
+  },
+}) => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    navigation.setOptions({ title: null });
+  });
```

Comment:
un `useEffect` sin dependencias hace que se corra cada vez que se renderiza el componente.

```suggestion
  useEffect(() => {
    navigation.setOptions({ title: null });
  }, [navigation]);
```